index.html

```diff
@@ -1680,9 +1680,9 @@ <h3>Status</h3>
 
         <p>
 Defining the data model, formats, and protocols for status schemes are out of
-scope for this specification. A status scheme registry [[?VC-STATUS-REGISTRY]]
-exists for implementers who want to implement <a>verifiable credential</a>
-status checking.
+scope for this specification. A status scheme registry
```

Comment:
not sure of the best way to re-word this, or if it is entirely necessary to do so, but this sentence calls the "extension registry" a "status scheme registry" and that disturbs by pedantic sensibilities.

Reply:
Yes, good catch. Fixed in 18d32bec5a90ec2b6c056b6d94695dcbba3094a9.